src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1450,23 +1450,12 @@ private BoundExpression BindAssignment(AssignmentExpressionSyntax node, BindingD
                 return BindDeconstruction(node, diagnostics);
             }
 
-            BindValueKind lhsKind;
-            ExpressionSyntax rhsExpr;
-            bool isRef = false;
+            var rhsExpr = node.Right.CheckAndUnwrapRefExpression(diagnostics, out var refKind);
+            var isRef = refKind == RefKind.Ref;
+            var lhsKind = refKind == RefKind.Ref ? BindValueKind.RefAssignable : BindValueKind.Assignable;
```

Comment:
is it simpler to just read `isRef` here?

Reply:
yes :D

Reply:
done.